packages/cli/vim/config/.vimrc

```diff
@@ -28,7 +28,7 @@ set scrolloff=8                 "Makes cursor stay 8 lines away from the top or
 
 
 "Tabs to spaces
-set tabstop=4 shiftwidth=4 noexpandtab
+set tabstop=4 shiftwidth=4 expandtab
```

Comment:
I realize my solution shouldn't work.... but it does on my machine. I don't do enough python to care anymore. I'll pull as-is